app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsDiffDetailView.kt

```diff
@@ -0,0 +1,30 @@
+package org.wikipedia.suggestededits
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import android.widget.LinearLayout
+import androidx.annotation.DrawableRes
+import kotlinx.android.synthetic.main.view_suggested_edits_contribution_diff_detail.view.*
+import org.wikipedia.R
+
+class SuggestedEditsDiffDetailView constructor(context: Context, attrs: AttributeSet? = null) : LinearLayout(context, attrs) {
+
+    init {
+        LayoutInflater.from(context)
+                .inflate(R.layout.view_suggested_edits_contribution_diff_detail, this, true)
+    }
+
+    fun setLabelAndDetail(labelText: String?, detailText: String?, @DrawableRes drawableRes: Int) {
```

Comment:
You may able to set the default value to the parameters so that we will no longer need to send empty value or `-1` to them.

Something like: `setLabelAndDetail(labelText: String? = "", detailText: String? = "", @DrawableRes drawableRes: Int = -1)`

Reply:
and to the settings above you can simply do `language DetailView.setLabelAnd Detail()`